torchtext/transforms.py

```diff
@@ -327,20 +327,43 @@ def _tokenize(self, text: str) -> List[str]:
 
         return bpe_tokens
 
-    def forward(self, input: Any) -> Any:
+    @torch.jit.export
+    def _tokenize(self, text: str) -> List[str]:
+        """Tokenize text into a list of tokens
+
+        Args:
+            text: An input text string.
+
+        Returns:
+            A list of bpe token ids represents each bpe tokens
+
+        For example: "awesome,awe"
+            --> bpe --> bpe tokens: ["aw", "esome"], [","], ["aw", e]
+        """
+        return self.bpe.tokenize(text)
+
+    def forward(self, input: Any, return_tokens: bool = False) -> Any:
```

Comment:
Just wondering if you think it makes more sense to pass this `return_tokens` flag into the `GPT2BPETokenizer` constructor instead? I would think that after creating a instance of this tokenizer, the user will consistently want to retrieve token token IDs or tokens instead of switching between the two options.

Reply:
hmm, it's a good point. That way we can keep the forward API cleaner as well :)